.github/workflows/ci-security-scan.yml

```
@@ -55,9 +55,10 @@ jobs:
           
           # Node.js build 
           cd ../../webapp-react
-          if [ ! -d "node_modules" ]; then
-            npm ci --prefer-offline --no-audit --progress=false
-          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)